judgments_admin/views.py

```diff
@@ -0,0 +1,48 @@
+from django.http import Http404, HttpResponse
+from django.template import loader
+from lxml import etree
+
+from judgments.api_client import (
+    MarklogicAPIError,
+    MarklogicResourceNotFoundError,
+    api_client,
+)
+
+akn_namespace = {"akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0"}
+
+
+def edit(request, judgment_uri):
+    try:
+        judgment_xml = api_client.get_judgment_xml(judgment_uri)
+        xml = etree.XML(bytes(judgment_xml, encoding="utf8"))
+        name = xml.xpath(
+            "//akn:FRBRname/@value",
+            namespaces=akn_namespace,
+        )[0]
+        context = {"metadata_name": name, "uri": judgment_uri}
+    except MarklogicResourceNotFoundError:
+        raise Http404("Judgment was not found")
+    template = loader.get_template("judgments_admin/edit.html")
+    return HttpResponse(template.render({"context": context}, request))
+
+
+def update(request):
+    context = {}
+    try:
+        judgment_xml = api_client.get_judgment_xml(request.POST["uri"])
+        xml = etree.XML(bytes(judgment_xml, encoding="utf8"))
+        name = xml.xpath(
+            "//akn:FRBRname",
+            namespaces=akn_namespace,
+        )[0]
+        name.set("value", request.POST["metadata_name"])
+        api_client.save_judgment_xml(request.POST["uri"], xml)
+        context["metadata_name"] = xml.xpath(
+            "//akn:FRBRname/@value",
+            namespaces=akn_namespace,
+        )[0]
```

Comment:
It's early days, so this is probably fine... Just wondered if it's worth thinking about what it the responsibility of the api client, and what is the responsibility of the view. Maybe there's no need for the views to know anything about XML or the underlying structure? If that's a detail of how the client interacts with marklogic, maybe that should be taking care of value prep?